### DISCUSSION TITLE
Thank you for making this! <3

### DISCUSSION THREAD

**Discussion by imaperson1060** [shaked6540/YoutubePlaylistDownloader#432] [General]
I came across this a few years ago while trying to make a YouTube downloader of my own, and while I do still use mine for single videos here and there, this is my go to app for playlists and batch downloading. Also, I _hated_ MetroUI on Windows 8, but this app shows exactly how it should've been done, rather than the clunky Microsoft apps. ⭐⭐⭐⭐⭐

All the people hating on this project just because you support Israel can screw themselves. This project is awesome!! (it's so funny seeing how you renamed all their issue titles lmao)

**Comment by shaked6540** [comment]
Thanks, it feels good to get compliments :)
